web/spacewalk-web.spec

```diff
@@ -164,6 +179,16 @@ install -m 644 conf/rhn_dobby.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-de
 install -m 755 modules/dobby/scripts/check-database-space-usage.sh $RPM_BUILD_ROOT/%{_sysconfdir}/cron.daily/check-database-space-usage.sh
 install -m 0644 etc/sysconfig/SuSEfirewall2.d/services/susemanager-database %{buildroot}/%{_sysconfdir}/sysconfig/SuSEfirewall2.d/services/
 
+%{__mkdir_p} %{buildroot}/srv/www/htdocs/javascript/manager
+cp -r html/src/dist/javascript/manager %{buildroot}/srv/www/htdocs/javascript/manager
+
+%{__mkdir_p} %{buildroot}/srv/www/htdocs/vendors
+cp html/src/dist/vendors/vendors.bundle.js %{buildroot}/srv/www/htdocs/vendors/vendors.bundle.js
+
+%files -n susemanager-web-libs
+%dir %{www_path}/www/htdocs/vendors
```

Comment:
Not sure if we should add a defattr() here. With new versions of RPM it is not needed anymore, but we have rpmlint running which tell us that it is better to have it.

Maybe you just add
```
%defattr(-,root,root,-)
```

to be on the save side.